golem-cluster.yaml

```
@@ -36,29 +36,34 @@ provider:
         outbound_urls: ["https://pypi.dev.golem.network"]
 
         # Minimal values of parameters offered by Golem provider 
-        min_mem_gib: 0
-        min_cpu_threads: 0
-        min_storage_gib: 0
-
-      budget_control:
-        # Estimated expected load and duration for worker that tells budget control to pick the least expensive Golem provider offers first.
-        # If not provided, offers will be picked at random.
-        # Both values need to be defined or undefined together.
-        expected_cpu_load: 0.8
-        expected_duration_minutes: 20
-
-        # Amount of GLMs for expected usage which Golem provider offer will be rejected if exceeded.
-        # Requires "expected_cpu_load" and "expected_duration_minutes" parameters to take effect.
-        max_expected_usage_cost: 1.5
+        min_mem_gib: 1
+        min_cpu_threads: 1
+        min_storage_gib: 1
+
+        # Maximal values of parameters offered by Golem provider
+        max_cpu_threads: null
+
+      # Golem providers differ from each other
+      per_cpu_budget:
+        expected_usage:
+          # Estimated expected load and duration for worker that tells budget control to pick the least expensive Golem provider offers first.
+          # If not provided, offers will be picked at random.
+          # Both values need to be defined, undefined or nulled together.
+          cpu_load: 0.8
+          duration_hours: 0.5 # 30 minutes
+
+          # Amount of GLMs for expected usage which Golem provider offer will be rejected if exceeded.
+          # Requires "cpu_load" and "duration_hours" parameters to take effect.
+          max_cost: 1.5
 
         # Amount of GLMs for worker initiation which Golem provider offer will be rejected if exceeded.
         max_initial_price: 0.5
 
-        # Amount of GLMs for CPU utilisation per second which Golem provider offer will be rejected if exceeded.
-        max_cpu_sec_price: 0.0005
+        # Amount of GLMs for CPU utilisation per hour which Golem provider offer will be rejected if exceeded.
+        max_cpu_hour_price: 1.8
```

```suggestion
        max_cpu_hour_price: 0.5
```
1.8 seems weird, plus it is really big when looking at https://stats.golem.network/network/providers/online
0.5 is also too big - I would like us to play with the settings and see, but for now 0.5 seems ok